static/app/components/profiling/FlamegraphViewSelectMenu.tsx

```diff
@@ -0,0 +1,52 @@
+import Button from 'sentry/components/button';
+import ButtonBar from 'sentry/components/buttonBar';
+import {t} from 'sentry/locale';
+
+interface FlamegraphViewSelectMenuProps {
+  onSortingChange: (sorting: FlamegraphViewSelectMenuProps['sorting']) => void;
+  onViewChange: (view: FlamegraphViewSelectMenuProps['view']) => void;
+  sorting: 'call order' | 'left heavy';
+  view: 'top down' | 'bottom up';
+}
+
+function FlamegraphViewSelectMenu({
+  view,
+  onViewChange,
+  sorting,
+  onSortingChange,
+}: FlamegraphViewSelectMenuProps): React.ReactElement {
+  return (
+    <ButtonBar merged>
```

Comment:
Is the gif outdated? Doesn't look like it's using the button bar

Reply:
Yes, indeed, it's outdated!